syntax = "proto2";
import "game_commands.proto";
message Command_Concede {
    extend GameCommand {
        optional Command_Concede ext = 1017;
    }
}

message Command_Unconcede {
    extend GameCommand {
        optional Command_Unconcede ext = 1032;
    }
}